client/state/jetpack-connect/reducer.js

```diff
@@ -150,9 +147,22 @@ export function jetpackSSO( state = {}, action ) {
 	return state;
 }
 
+export function jetpackSSOSessions( state = {}, action ) {
+	switch ( action.type ) {
+		case JETPACK_CONNECT_SSO_AUTHORIZE_SUCCESS:
+			const parsedUrl = urlModule.parse( action.ssoUrl );
+			return Object.assign( {}, state, buildNoProtocolUrlObj( parsedUrl.hostname ) );
+		case SERIALIZE:
```

Comment:
We need the `serialize` and `deserialize` actions for persisting state between requests. 

https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#solution-serialize-and-deserialize-actions